identity/app/idapiclient/IdApi.scala

```diff
@@ -216,4 +219,55 @@ trait IdApiUtils {
       slug.stripPrefix("/").stripSuffix("/")
     }) mkString "/"
   }
+
+  def GET(uri: String, urlParameters: Parameters, headers: Parameters): Future[Response[HttpResponse]] = {
+    wsClient
+      .url(uri)
+      .withQueryStringParameters(urlParameters.toList: _*)
+      .withHttpHeaders(headers.toList: _*)
+      .get()
+      .map { resp => Right(HttpResponse(resp.body, resp.status, resp.statusText)) }
+      .recover {
+        case e: Throwable =>
+          logger.error(s"Network error while communicating with ${apiUrl("auth")}:", e)
+          Left(List(Error(e.getClass.getName, e.toString)))
```

Comment:
There's quite a lot of similarity between these methods.  Could we do something like:

```
def makeRequest(method: String, ...) {
   wsClient
      .url(uri)
      ...
      .execute(method)
      .map ...
}

def GET = makeRequest("GET", None, _, _)
def POST = makeRequest("POST", _, _, _)
```
Obviously there would need to be a little handling for the adding the body only if necessary.